scripts/dcommand.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
```

Comment:
Can you maybe add a docstring to this script describing what it's meant to do?

Reply:
👍